Aleksandr.Timoshenko/src/main/java/CalculatorScanner.java

```diff
@@ -0,0 +1,59 @@
+import java.util.Scanner;
+
+public class CalculatorScanner{
+
+
+
+ public static void main(String[] args){
+    int firstOperand = 0;
+    int secondOperand = 0;
+
+    Scanner sc = new Scanner(System.in);
+    System.out.println("Enter the operation");
+    String operation = sc.nextLine();

```

Comment:
Use sc.next() instead of sc.nextLine(). It will be waiting until your input then
and you don't have to check if (sc.hasNextInt())